keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -107,9 +113,15 @@ def __init__(self, img_height=512, img_width=512, jit_compile=False):
             origin="https://huggingface.co/fchollet/stable-diffusion/resolve/main/kcv_decoder.h5",
             file_hash="ad350a65cc8bc4a80c8103367e039a3329b4231c2469a1093869a345f55b1962",
         )
+        image_encoder_weights_fpath = keras.utils.get_file(
+            origin="https://huggingface.co/fchollet/stable-diffusion/blob/main/vae_encoder.h5",
+            file_hash="005390320d6095734833d0b442d11f413aae42c4d42872f3b352f7f53b5da122",
```

Comment:
it is now SHA256: 7ae08f918d4e3a686807474706a6821ca1ff3c71b84d98ba33081e6cb7aab9e3

Reply:
> OSError: Unable to open file (file signature not found)
The error message has a very low usability.

Reply:
Trying now with a manually specified hash algorithm.

Reply:
https://huggingface.co/fchollet/stable-diffusion/blob/main/vae_encoder.h5
![immagine](https://user-images.githubusercontent.com/1710528/193895372-67662b2b-8cfe-42e0-af1f-d5f627d86ee0.png)

Reply:
It doesn't match locally with wget:

```bash
sha256sum vae_encoder.h5 
c60fb220a40d090e0f86a6ab4c312d113e115c87c40ff75d11ffcf380aab7ebb`

md5sum vae_encoder.h5 
d3c5e2b8e4eee794417f9552fe0cb388  vae_encoder.h5
```

Reply:
Updated to work.  try clearing your python cache

Reply:
> Updated to work. try clearing your "~/.keras" cache

It is not the cache, It was commited `c60` 14 minutes ago..

Reply:
I know.  I just mean't try re-running now to confirm it works.